src/averageAges.js

```diff
@@ -20,6 +20,24 @@ function calculateMenAverageAge(people, century) {
   // avoid using loop and forEach
   // replace `if ()` statement with &&, || or ?:
   // without nesting
+  const mans = people.filter(person => person.sex === 'm');
```

Comment:
use century inside this filter
so you will find all men or if century is passed all men died in certain century
then you won't need if else